addons/payment/models/account_move.py

```diff
@@ -26,10 +27,18 @@ def get_portal_last_transaction(self):
         return self.with_context(active_test=False).transaction_ids._get_last()
 
     def payment_action_capture(self):
-        self.authorized_transaction_ids._send_capture_request()
+        if not (self.env.user.has_group('account.group_account_invoice')
+            and self.check_access_rights('write') and self.check_access_rule('write') is None):
```

Comment:
`check_access_rule` will : `:raise UserError: * if current ir.rules do not permit this operation.`, so, you won't get your AccessError in this case :)